guidelines/content-guidelines/02-diagrams.md

```diff
@@ -11,6 +11,17 @@ To convey the intended message effectively in a diagram, follow these basic prin
 
 For details on how to format diagrams and their elements in Kyma documents, see the particular document sections.
 
+## Reference
+
+Always add an alternative text that concisely describes the content or function of the diagram you are referring to. The alt text:
```

Comment:
```suggestion
Always add an alternative text that concisely describes the content or function of the diagram you are referring to. The alternative text:
```

Reply:
Alt text is another name for alternative text. As a compromise, I can introduce it first in parentheses, is that ok?